pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/rbac/role_binding.go

```diff
@@ -43,6 +44,11 @@ func (f *RoleBinding) SetTemplateDefaults() error {
 const managerBindingTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
+  labels:
+    app.kubernetes.io/name: manager-rolebinding
+    app.kubernetes.io/component: manager
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```
ihmo: The architecture component is RBAC or permissions  such as in the examples we have a database